README.md

```diff
@@ -1,6 +1,14 @@
 # Security Belts
 
-## Methodology
+With modern software development methodologies such as DevOps, development teams have the need to take care and also take responsibility for software security of their product themselves. In order to achieve this, the team has to deal with security early on and continuously in its software development process. To empower the teams to handle software security well they have to build up knowledge and awareness for the subject and are willing to continuously improve themselves. But also organizations have to deal with these new requirements and have to enable software security to function decentralized within their organization landscape.
```

Comment:
```suggestion
With modern software development methodologies such as DevOps, development teams have the need to take over responsibility for software security for their product themselves. To avoid security fire drills right before deployment, the team must deal with security early on and continuously in its software development process. To empower the teams to handle software security well, they must build up knowledge and awareness for the subject. However, even more fundamental is their commitment to continuously improve themselves. On an organizational level, it is crucial to enable software security to function decentralized across multiple teams.
```